src/Modules/ModuleTypes.jl

```diff
@@ -132,7 +132,7 @@ true
 ```
 """
 mutable struct FreeModElem{T} <: AbstractFreeModElem{T}
-  coords::SRow{T} # also usable via coeffs()
+  coords::SRow{T, Vector{T}} # also usable via coeffs()
```

Comment:
I think this won't work in general, as there are special implementations for the second type parameter, e.g. for ZZ. (please correct me if I am wrong @thofma)

Reply:
See https://github.com/thofma/Hecke.jl/blob/7d8744d1856450d4b1537a36d34e9b4776b92f9b/src/Sparse/ZZRow.jl#L192 and below

Reply:
Yes. One probably needs a `sparse_row_type(T)` method (which does not exist so far).

Reply:
Hrm, that's annoying. Then we need to make this type part of the signature for FreeModElem, which is annoying, as it is kind of an implementation detail. Maybe it suffices to ensure all access to `coords` is through an accessor function and have that add a type assertion?

But yeah it would still need a `sparse_row_type` method...

Reply:
I would go via the accessor method plus type assertion.

Reply:
Will do. I an also working on adding `sparse_row_type`